iaslic-bulletin.csl

```diff
@@ -0,0 +1,397 @@
+    <choose>
+      <if type="book article-newspaper entry-dictionary entry-encyclopedia" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else-if type="chapter paper-conference" match="any">
+        <text variable="title"/>
+      </else-if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title" font-style="italic"/>
+  </macro>
+  <macro name="issued">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="edition">
+    <text variable="edition" suffix=" ed."/>
+  </macro>
+  <macro name="in-book-iaslic">
+    <text value="In : " font-style="italic"/>
+    <names variable="editor" suffix=", ">
+      <name form="long" name-as-sort-order="all" delimiter=" and ">
+        <name-part name="family" text-case="uppercase"/>
+        <name-part name="given" prefix=" (" suffix=")"/>
+      </name>
+    </names>
+    <text term="editor" form="short" text-case="capitalize-first" font-style="italic"/>
+    <text variable="container-title" font-style="italic" prefix=" " suffix=". "/>
+    <text macro="volume"/>
+    <text macro="issued" prefix=" " suffix=". "/>
+    <text macro="edition" suffix=". "/>
+    <text macro="publisher"/>
+    <text macro="pages" prefix=" "/>
+  </macro>
+  <macro name="publisher">
+    <group delimiter="; ">
+      <text variable="publisher"/>
+      <text variable="publisher-place" suffix=","/>
+    </group>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if variable="number-of-pages">
+        <text variable="number-of-pages" prefix="p" suffix="."/>
+      </if>
+      <else>
+        <text variable="page" prefix="p" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="volume">
+    <label variable="volume" form="short"/>
+    <text variable="volume" suffix="."/>
+  </macro>
+  <citation>
+    <layout delimiter=", ">
+      <group vertical-align="sup">
+        <text value="["/>
+        <text variable="citation-number"/>
+        <text value="]"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="10" et-al-use-first="6" name-as-sort-order="all" sort-separator=" " entry-spacing="0" line-spacing="2">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout>
+      <group delimiter=". ">
+        <text variable="citation-number"/>
+        <text macro="author"/>
+        <choose>
+          <if type="book">
+            <text macro="title" font-style="italic"/>
+            <text macro="volume"/>
+            <text macro="issued"/>
+            <text macro="edition"/>
+            <group delimiter=", ">
+              <text macro="publisher"/>
+              <text macro="pages"/>
+            </group>
+          </if>
+          <else-if type="entry-dictionary">
+            <group delimiter=" ">
+              <text macro="title" font-style="italic" suffix=". "/>
```

Comment:
all these dot suffixes would be better set on the group level. Please look over this.